src/cdk/text-field/_text-field.scss

```diff
@@ -4,8 +4,18 @@
   // by watching for the animation events that are fired when they start. Note: the /*!*/ comment is
   // needed to prevent LibSass from stripping the keyframes out.
   // Based on: https://medium.com/@brunn/detecting-autofilled-fields-in-javascript-aed598d25da7
-  @keyframes cdk-text-field-autofill-start {/*!*/}
-  @keyframes cdk-text-field-autofill-end {/*!*/}
+  @keyframes cdk-text-field-autofill-start {
+    0% {
```

Comment:
please put `/*!*/` in the empty blocks to prevent them from being stripped out by sass